ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py

```diff
@@ -1743,6 +1744,61 @@ def test_jax_lax_dot(
     )
 
 
+@st.composite
+def _get_dtype_inputs_for_batch_matmul(draw):
+    lhs_shape = draw(nph.array_shapes(min_dims=2, max_dims=6, min_side=1, max_side=4))
+    rhs_shape = list(lhs_shape)
+    rhs_shape[-1], rhs_shape[-2] = rhs_shape[-2], rhs_shape[-1]
+    rhs_shape = tuple(rhs_shape)
+    dtype = draw(helpers.get_dtypes("valid", index=1, full=False))
+    lhs = draw(
+        helpers.array_values(
+            dtype=dtype[0],
+            shape=lhs_shape,
+            min_value=2,
+            max_value=5,
+        )
+    )
+    rhs = draw(
```

Comment:
You are overcomplicating things a bit. You can just use the `dtype_and_values` helper which does not require manually sampling a dtype and a shape beforehand. This way you could just do:
```
   dtype, lhs = draw(
        helpers.dtype_and_values(
            min_value=2,
            max_value=5,
        )
    )
   lhs_shape = lhs.shape
```